microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md

```diff
@@ -0,0 +1,180 @@
+
+You can see your web project in a dedicated tab inside of Visual Studio Code, and also simulate various devices:
+
+:::image type="content" source="../../media/2021/11/edge-devtools-for-vscode-simulated-iphone-red-boxes.png" alt-text="The screencast showing the current document in an emulated iPhone 5 in the correct size and with an emulated touchscreen interface.":::
+
+<!-- videos: see work item -->
+
+For details about the dockable screencast, see [Changing to headless mode](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#changing-to-headless-mode).  For details about device emulation, see [Device emulation in the screencast](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#device-emulation-in-the-screencast).
+
+### Live, inline issue reporting
+
+The extension also now features live, inline issue reporting.  Instead of finding out about issues in a separate tool, Visual Studio Code highlights the issues in your source code and reports them live while you enter your code:
+
+:::image type="content" source="../../media/2021/11/edge-devtools-for-vscode-inline-issue-reporting-addl-red.png" alt-text="An accessibility issue reported inside a piece of code showing how to fix the problem, and where to find more information.":::
+
+<!-- videos: see work item -->
+
+For details, see [Inline and live issue analysis](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#inline-and-live-issue-analysis).
+
+
+<!-- ====================================================================== -->
+## Sources tool now notifies you when sourcemaps can't be loaded
+
+<!-- 4 -->
+
+<!-- Title: Get notified when DevTools cannot load your sourcemaps correctly -->
+<!-- Subtitle: The Sources tool now provides several places in the UI when DevTools can't fetch or parse your sourcemaps. -->
+
+In Microsoft Edge version 96, the **Sources** tool now provides several indications in the UI when DevTools cannot load your sourcemaps.  In the **Page** tab in the **Navigator** pane in the **Sources** tool, files that DevTools can't load sourcemaps for have a warning<!--todo: capture/show--> icon in place of the file<!--todo: capture/show--> icon.  
+
+Selecting a file with the warning icon opens the file in the Sources tool with an infobar<!--todo: capture/show--> indicating that DevTools either couldn't fetch the sourcemap correctly from the server or couldn't parse the sourcemap correctly.
+
+From the infobar, you can learn more about the issue by selecting the **Open in Issues** button.  The **Issues** tool then opens in the **Drawer** and provides information about how to resolve the issue and get your sourcemap loaded correctly in DevTools.
+
+* To prevent the infobar about sourcemaps from cluttering the **Sources** tool, select the **Don't show again** button.
```

Comment:
I'm not really sure these 2 lines make sense as bullets in a list. I propose making these 2 sentences in one paragraph and then moving it down below the image. It's not must-have information but it is nice-to-have since we're not sure if this will cause undue clutter for users.

To clarify what I'm proposing: I'd add "To learn more about how DevTools fetches and parses sourcemaps..." as the last sentence in the paragraph.